examples/basics/visuals/colorbar_visual_types.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+
+"""Using Colorbars with the Canvas"""
+
+from vispy import app
+from vispy import gloo
+from vispy.visuals import ColorBarVisual
+from vispy.color import get_colormap
+from vispy.visuals.transforms import NullTransform
+from vispy.visuals.transforms.transform_system import TransformSystem
+
+colormap = get_colormap("ice")
+
+
+def style_colorbar(colorbar):
+    colorbar.label.font_size = 18
+    colorbar.label.color = "black"
+
+    colorbar.clim = (0, 10)
+
+    colorbar.ticks[0].font_size = 14
```

Comment:
setting ticks needs you to access them individually. This should change once ```AxisVisual``` stabilizes, but until then, is there anyway to make this nicer? (overload the ```=``` operator by returning a custom ```Ticks``` class came to my mind, but it felt like over-engineering)